packages/wrangler/src/workflows/commands/instances/describe.ts

```diff
@@ -0,0 +1,279 @@
+import { logRaw } from "@cloudflare/cli";
+import { red, white } from "@cloudflare/cli/colors";
+import {
+	addMilliseconds,
+	formatDistanceStrict,
+	formatDistanceToNowStrict,
+} from "date-fns";
```

Comment:
Nit: We might be able to implement most of these with the [Intl.RelativeTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat) API to avoid an extra dependency.

Reply:
Good shout. Let's follow up with this. I want to remove both `itty-time` and `dates-fns` in another PR

Reply:
Yes I agree! cc @sidharthachatterjee please add this to TODOs for post launch